tests/integration/test_examples.py

```diff
@@ -0,0 +1,80 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import shutil
+from pathlib import Path
+
+import pytest
+
+from zenml.cli import EXAMPLES_RUN_SCRIPT, LocalExample
+
+QUICKSTART = "quickstart"
+NOT_SO_QUICKSTART = "not_so_quickstart"
+CACHING = "caching"
+
+
+@pytest.fixture(scope="session")
+def examples_dir(tmp_path_factory):
+    tmp_path = tmp_path_factory.mktemp("tmp")
+    examples_path = tmp_path / "zenml_examples"
+    source_path = Path("examples")
+    shutil.copytree(source_path, examples_path)
+    yield examples_path
+    shutil.rmtree(str(examples_path))
+
+
+def test_run_quickstart(examples_dir: Path):
+    """"""
```

Comment:
I think we should either have a small docstring here or just remove the quotes entirely (same for the other functions)

Reply:
I must have gotten distracted when I wanted to write them